src/components/calendar/calendar.ts

```diff
@@ -141,6 +141,9 @@ export class SbbCalendarElement<T = Date> extends LitElement {
   /** A function used to filter out dates. */
   @property({ attribute: 'date-filter' }) public dateFilter?: (date: T | null) => boolean;
 
+  /** A specific date for the current datetime (timestamp in milliseconds). */
```

Comment:
Would it not be possible to have a mixin for the now prop? There you could use a getter and setter and therefore encapsulate the parsing logic into the mixin. consuming classes could just read this.now returning a Date. maybe a second getter for just checking if there is date. but maybe not really necessary.

Reply:
Mixin added, to be reviewed

I also noticed that the timetable-related components use the logic 'take the `now` value or use Date.now()', so i added a specific getter for this; a similar logic is also used in clock and datepicker.